File: StudentCard.Application/Students/Dtos/PersonLotDtos/PersonDoctoralDtos/PersonDoctoralDto.cs
using StudentCard.Application.Students.Dtos.PersonLotDtos.BaseDtos;
using StudentCard.Data.Rdpzsd.Parts.Base;
using System;
using System.Collections.Generic;

namespace StudentCard.Application.Students.Dtos.PersonLotDtos.PersonDoctoralDtos
{
    public class PersonDoctoralDto : BaseStudentDoctoralDto
    {
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }

        public RdpzsdAttachedFile PeRecognitionDocument { get; set; }

        public List<DoctoralSemesterDto> Semesters { get; set; } = new List<DoctoralSemesterDto>();
    }
}

File: WebApplication/src/modules/student/dtos/person-lot.dtos/person-doctoral.dtos/person-doctoral.dto.ts
import { BaseStudentDoctoralDto } from "../base.dtos/base-student-doctoral.dto";
import { DoctoralSemesterDto } from "./doctoral-semester.dto";

export class PersonDoctoralDto extends BaseStudentDoctoralDto {
  startDate: Date;
  endDate: Date;
  semesters: DoctoralSemesterDto[];
}